### DISCUSSION TITLE
Invalid verification token throws a 200

### DISCUSSION THREAD

**Discussion by mattsbanner** [craftcms/cms#10592] [Ideas] [answered]
### Description
A `HttpException` [gets thrown](https://github.com/craftcms/cms/blob/535a86d34ec4eb3fb8a6f038f3b3d5e09dd3a8bc/src/controllers/UsersController.php#L2171) with a 200 status code when a user provides an invalid password reset token.

This causes plugins like [Sentry SDK](https://plugins.craftcms.com/sentry-sdk) to post the error to Sentry.

It feels to me like this should be a `BadRequestHttpException` where a 400 is thrown, allowing us to ignore it in the plugin config.

### Additional info
- Craft version: 3.7.28
- PHP version: 7.4
- Database driver & version: MySQL 5.7
- Plugins & versions: Sentry SDK 1.1.2

**Comment by brandonkelly** [comment]
This could be a breaking change, so converted it to a discussion.

Related: #8631

**Comment by timkelty** [comment, marked as answer]
This is [fixed in Craft 4](https://github.com/craftcms/cms/commit/43acd383adc817b98ff64374b706737165b3a936).
Thanks, @mattsbanner!
